src/Header/Bookmark.js

```diff
@@ -0,0 +1,31 @@
+import React, { useEffect, useState } from 'react'
+
+export default function Bookmark(){
+  const [folderData, setfolderData] = useState('');
+
+  useEffect(()=>{
+    async function folderDatas(){
+      try{
+        const response = await fetch('https://bootcamp-api.codeit.kr/api/sample/folder');
+        const data = await response.json();
+        const folder = data.folder; // response 안에 있는 folder 데이터
+        setfolderData(folder) // folder 데이터를 folderData에 저장
+      }
+      catch(error){
+        console.log('Error')
+      }
+    }
+    folderDatas()
+  })
+  return (
+    <div className='headerUnder'>
+    {folderData && folderData.owner && ( // folderData가 있고, folderData 안에 owner가 있으면
```

Comment:
[중요도: 중간]
```suggestion
    {folderData?.owner &&
```
주석을 적어주셔서 더 빠르게 파악이 됬네요.
optionalChaining이라는 것을 사용한다면 굳이 folderData가 있는지 그리고 folderData의 owner가 있는지를 축약해서 사용할 수 있습니다!

Reply:
optionalChaining... 리마인드 감사합니다!!https://github.com/codeit-bootcamp-frontend/3-Weekly-Mission/pull/185/commits/12de3aa04b5f4b42a8ef6a6f39d73781c0d142bb